comps/comps-foreman-plugins-rhel7.xml

```diff
@@ -45,6 +45,7 @@
       <packagereq type="default">ruby193-rubygem-hammer_cli_foreman_tasks</packagereq>
       <packagereq type="default">ruby193-rubygem-ovirt_provision_plugin</packagereq>
       <packagereq type="default">ruby193-rubygem-puppetdb_foreman</packagereq>
+      <packagereq type="default">rubygem-algebrick</packagereq>
```

Comment:
It's not clear, but the upper group is top-level packages (plugins) while the bottom set, where algebrick should be, is for dependencies.

Also run ./comps_doc.sh and it'll add the -doc subpackage.